DFE-Digital/early-careers-framework
# [CPDLP-1236] Add ECF statement selector to finance area

## Ticket and context

Ticket: https://dfedigital.atlassian.net/browse/CPDLP-1236

- Added statement selector to ECF statements in finance area
- Also tweaked the top area of the page to match the designs more

## Screenshots

![Screenshot 2022-04-13 at 09 40 01](https://user-images.githubusercontent.com/92580/163139706-e5b3cd5f-32eb-49cf-aa06-2dd6ee4b893e.png)

## Tech review

### Is there anything that the code reviewer should know?

### Code quality checks

### HTML Checks

### Gotchas

## Product review

### How can someone see it it review app?

- Login as a finance user
- View an ECF statement
- Should be able to change statements now via the statement selector component

## External API changes

- None
